addons/stock_picking_wave/controllers/main.py

```diff
@@ -1,16 +1,15 @@
-from openerp.addons.web import http
-from openerp.addons.web.http import request
+from openerp import http
+from openerp.http import request
 
 
 class picking_wave_report(http.Controller):
     @http.route('/report/stock_picking_wave.report_pickingwave/<ids>', type='http', auth='user', 
```

Comment:
Note: It seems there isn't any specific option available in official modules to call this controller, this controller may be removed while merging.